src/electionguard_cli/cli_models/import_ballots/import_ballot_inputs.py

```diff
@@ -0,0 +1,29 @@
+from typing import List
+from electionguard.ballot import SubmittedBallot
+from electionguard.election import CiphertextElectionContext
+from electionguard.guardian import Guardian
+from electionguard.manifest import Manifest
+from electionguard_cli.cli_models.shared.cli_election_inputs_base import (
+    CliElectionInputsBase,
+)
+
+
+class ImportBallotInputs(CliElectionInputsBase):
```

Comment:
For what it's worth, usually these models would be placed in the same module as the item using them. This seperation feels very C#.

```py
from random_step import RandomStepModel, random_step_method
```

Reply:
The three top level folders in CLI are currently:

- `cli_models`
- `commands`, and
- `steps`

Just to confirm are you proposing eliminating the `cli_models` folder entirely and moving all of those objects into the `steps` folder?  That seems odd to me since the models are used by both commands and steps.  But if that's pythonic I'll certainly do it.

However, this case is especially difficult because ImportBallotInputs is used by classes in both `/steps/import_ballots`, and `steps/shared`.  Which of those two folders would you have me move this particular class into?